files/provisioning.py

```diff
@@ -63,7 +63,14 @@ def _generatePlaybook(self, kernel_desc,initrd_desc, preseed_name,
               timeout: 3600
 
       - name: If the host's SSH is up, we can wipe the root password
-        shell: 'passwd -l root'"""
+        shell: 'passwd -l root'
+
+      - name: Now let's set the hostname
+        shell: 'hostnamectl set-hostname "{{ mname }}"'
+
+      - name: And reboot because Linux can't handle hostname changing
```

Comment:
usually, if you just use "hostname foo" and log out, the next login will have it changed.

Reply:
yes but is it picked up by the networking stack ? It's mainly an Open HPC issue, I suspect it's relying heavly on hostnames being set.